docs/skopeo-standalone-sign.1.md

```diff
@@ -25,12 +25,12 @@ $ skopeo standalone-sign busybox-manifest.json registry.example.com/example/busy
 $
 ```
 
-## SEE ALSO
-skopeo(1), skopeo-copy(1), containers-signature(5)
-
 ## NOTES
 
-As described in [Container signature format](https://github.com/containers/image/blob/main/docs/containers-signature.5.md), this command is intended for use with local signatures, such as OpenPGP. Further, this command does **not** interact with the artefacts generated by [Docker Content Trust (DCT)](https://docs.docker.com/engine/security/trust/).
+This command is intended for use with local signatures, such as OpenPGP, as per containers-signature(5). Furthermore, this command does **not** interact with the artifacts generated by Docker Content Trust (DCT).
```

Comment:
Could you leave link to the containers-signature page, so that users could easily get it if looking on line?

[containers-signature(5)](https://github.com/containers/image/blob/main/docs/containers-signature.5.md)

Reply:
Sure, will do it shortly 🙇‍♀️

Reply:
Done @rhatdan thanks for the kind feedback 🙇